setup/src/create_db.py

```diff
@@ -230,6 +242,12 @@ def create_super_user():
     new_admin.first_name=os.getenv('HEAD_ORG_FIRST_NAME')
     new_admin.last_name=os.getenv('HEAD_ORG_LAST_NAME')
 
+def create_teams():
```

Comment:
loop through objects that have a team name and a team leaders instead of creating separate objects.